rapidfuzz/distance/Levenshtein.impl

```diff
@@ -486,14 +486,14 @@ Editops recover_alignment(InputIt1 first1, InputIt1 last1, InputIt2 first2, Inpu
         return editops;
     }
 
-    int64_t col = len1;
-    int64_t row = len2;
+    auto col = len1;
+    auto row = len2;
 
     while (row && col) {
-        uint64_t col_pos = col - 1;
-        uint64_t col_word = col_pos / 64;
+        std::size_t col_pos = col - 1;
+        std::size_t col_word = col_pos / 64;
         col_pos = col_pos % 64;
-        uint64_t mask = 1ull << col_pos;
+        std::size_t mask = static_cast<std::size_t>(1) << col_pos;
```

Comment:
This is really used as a uint64_t mask below:
```
if (matrix.VP[row - 1][col_word] & mask) {
```
and can be shifted by up to 63:
```
std::size_t col_word = col_pos / 64;
```
so size_t is incorrect here.